packages/console-client-wc/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
Can we put it in a test folder? Or name it to understand that it is a test?

Reply:
Running `vite preview` makes this file immediately available to the developer, any changes to the name or position will result in further things the developer needs to know to play around with the webcomponent.

I'd not change this until we have proper testing in place. I've added a short README file explaining the purpose of the index.html file.